docs/man_bodhi.rst

```diff
@@ -243,6 +243,58 @@ The ``updates`` command allows users to interact with bodhi updates.
 
     Request that the given update be changed to the given state. ``update`` should be given by
     update id, and ``state`` should be one of testing, stable, unpush, obsolete, or revoke.
+    
+    
+    
+Examples
+========
+
+Some common examples of bodhi CLI operations:
+
+    ``$ bodhi --new --type bugfix bodhi-0.4.2-1.fc7``
+        
+        Submitting a new update
```

Comment:
I think it would be more obvious if this text was above the example, and if the example were indented instead of this text being indented. As it is, some readers might think this text is a label for the line below it.